dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/rabbitmq.properties

```diff
@@ -27,4 +27,9 @@ mailqueue.view.bucketCount=1
 # Too little value will lead to unnecessary reads. Too big value will lead to more expensive browse.
 # Choose this parameter so that it get's update one time every one-two sliceWindow
 # Optional, default 1000
-mailqueue.view.updateBrowseStartPace=1000
\ No newline at end of file
+mailqueue.view.updateBrowseStartPace=1000
+
+# Enables or disables the gauge metric on the mail queue size
+# Choose to disable it when the mail queue size is getting too big
```

Comment:
Maybe we can add `computing the size of the mail queue is currently implemented on top of browse operation and thus have a linear complexity`
Also `metrics get exported periodically as configured in elasticsearch.properties, thus getSize is also called periodically`